text/0000-project-descriptor-images.md

```diff
@@ -0,0 +1,159 @@
+# Meta
+[meta]: #meta
+- Name: Support specifying images in project descriptor
+- Start Date: 2021-10-26
+- Author(s): [@phil9909](https://github.com/phil9909), [@modulo11](https://github.com/modulo11), [@pbusko](https://github.com/pbusko), [@jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Add a new array of tables `[[images]]` to the `project.toml` file.
+
+# Definitions
+[definitions]: #definitions
+
+* **project descriptor**: The `project.toml` file that describes a project.
+* **image**: the output of a running a buildpack(s) against a project
+* **image name**: Global Unique Name (GUN) of an OCI image in the form `[REGISTRY_HOST[:REGISTRY_PORT]/]REPOSITORY[:TAG]`.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, there is no support for specifying an output image in the `project.toml` although flags like `--tag` are already present.
```

Comment:
Is the use case to do a single build but want to be able to push to one or registries with the same image?